.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,15 @@
 - [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
 - [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.
 
+#### For EI only PR
```

Comment:
nit: Maybe we can change to "EIA Checklist"

Reply:
Updated. Thanks